src/shared/styles/_variables.scss

```diff
@@ -1,5 +1,5 @@
-$content-width: 600px;
-$min-width-container: 600px;
+$content-width: 1200px;
+$min-width-container: 1200px;
```

Comment:
don't change this, this change will affect other pages. You create a new one for the contest page like `$contest-width`